x-pack/test/functional/apps/observability_log_explorer/flyout_highlights.ts

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const DATASET_NAME = 'flyout';
+const NAMESPACE = 'default';
+const DATA_STREAM_NAME = `logs-${DATASET_NAME}-${NAMESPACE}`;
+const NOW = Date.now();
+
+const sharedDoc = {
+  time: NOW + 1000,
+  logFilepath: '/flyout.log',
+  serviceName: 'frontend-node',
+  datasetName: DATASET_NAME,
+  namespace: NAMESPACE,
+  message: 'full document',
+  logLevel: 'info',
+  traceId: 'abcdef',
+  hostName: 'gke-edge-oblt-pool',
+  orchestratorClusterId: 'my-cluster-id',
+  orchestratorClusterName: 'my-cluster-id',
+  orchestratorResourceId: 'orchestratorResourceId',
+  cloudProvider: 'gcp',
+  cloudRegion: 'us-central-1',
+  cloudAz: 'us-central-1a',
+  cloudProjectId: 'elastic-project',
+  cloudInstanceId: 'BgfderflkjTheUiGuy',
+  agentName: 'node',
+};
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const dataGrid = getService('dataGrid');
+  const testSubjects = getService('testSubjects');
+  const PageObjects = getPageObjects(['observabilityLogExplorer']);
+
+  describe('Flyout highlight customization', () => {
```

Comment:
How i see the tests as -
1. Flyout.ts - To handle loading of the flyout with the basic header level messages
2. flyout_highlights.ts - To load all highlights levels tests. They might grow with time.
3. flyout_ai_assisstant.ts - To handle AI stuff.

Why i see them like this -
1. This would provide us with a cleaner separation of concern and more focussed tests. Going forward we will be adding more features to this flyout and this separation would help us now.
2. Easily runnable on local environment rather than running 1 big flyout.tsx, we can run individual ones.

I mean we can have a folder called `flyout` and then move all these files there if its fine, but merging them into 1 huge file would be cumbersome.